tests/VObject/CliTest.php

```diff
@@ -483,7 +483,13 @@ public function testRepair()
         );
 
         rewind($this->cli->stdout);
-        $this->assertRegExp("/^BEGIN:VCARD\r\nVERSION:2.1\r\nUID:[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\r\nEND:VCARD\r\n$/", stream_get_contents($this->cli->stdout));
+        $regularExpression = "/^BEGIN:VCARD\r\nVERSION:2.1\r\nUID:[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\r\nEND:VCARD\r\n$/";
+        $data = stream_get_contents($this->cli->stdout);
+        if (method_exists($this, 'assertMatchesRegularExpression')) {
```

Comment:
maybe we coud do a regular preg_match() instead and assert the return value?

Reply:
The asserts that come with `phpunit` are nice. It is just this transition thing that is a bit annoying - the warnings start coming too quickly making it difficult to have a unit test code base that works with many phpunit major versions.

Reply:
I added a "ToDo" so that when someone comes across this in 2 years time, they will easily be confident to clean it up.

Reply:
@staabm are you OK with the ToDo and using the the "built-in" assert methods?
Or do you want me to use a "manual" `preg_match` plus assert result?